ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -84,38 +84,6 @@ public enum TableViewCellBackgroundStyleType: Int {
     }
 }
 
-// MARK: - Table Colors
-
-public extension Colors {
-    internal struct Table {
-        struct Cell {
-            static var image: UIColor = iconSecondary
-            static var title: UIColor = textPrimary
-            static var subtitle: UIColor = textSecondary
-        }
-
-        struct HeaderFooter {
-            static var accessoryButtonText: UIColor = textSecondary
-            static var background: UIColor = .clear
-            static var backgroundDivider: UIColor = surfaceSecondary
-            static var text: UIColor = textSecondary
-            static var textDivider: UIColor = textSecondary
-            static var textLink = UIColor(light: Palette.communicationBlueShade10.color, dark: communicationBlue)
-        }
-
-        static var background: UIColor = surfacePrimary
-        static var backgroundGrouped = UIColor(light: surfaceSecondary, dark: surfacePrimary)
-    }
-
-    // Objective-C support
-    @objc static var tableBackground: UIColor { return surfacePrimary }
```

Comment:
Have we double checked that this isn't being used in any downstream dependencies?

Reply:
These colors were used in a bunch of devmain files. The plan is to update those usages when porting fluent 2 colors into devmain.